assets/js/artist.js

```diff
@@ -164,12 +157,19 @@ searchBtnEl.click(() => {
         "method": "GET"
     };
     $.ajax(thisSearch).then(function (artistInfo) {
-        renderArtistPage(artistInfo);
+        artistInfo = artistInfo.artists[0];
```

Comment:
Maybe still try to abstract this to the `init()` function. Set `save` parameter to a localStorage entity?

Reply:
I started there and tried that but it wasn't working. Let me put it back though, because that was before I realized that some of the random artists themselves were not working.